index.html

```diff
@@ -18,8 +18,8 @@
         <ul class="hamburger-menu" id="main-menu">
           <li><a href="#">Home</a></li>
           <li><a href="#">Shop</a></li>
-          <li><a href="#">Wine Pairing</a></li>
-          <li><a href="#">Recipe Generator</a></li>
+          <li><a href="page-1.html">Wine Pairing</a></li>
+          <li><a href="page-2.html">Recipe Generator</a></li>
```

Comment:
Hi Gwen thank you for this. I think we should have a meeting to recap/agree on the functionalities of our app based on the API that I checked so far to be sure we are on the same page. So far an my idea to easily use the API, is to have:
- homepage: choice between red or white
- page "wines" (would now be a page where the content is updated via JS depending on whether the user chooses red or white. This page has a select menu (dropdown) that I have already developed. For each case (red or white) the select dropdown is populated by an array of wines that I have created by taking some of the wines available in the API. Once the user clicks one of the wines, we can get some information about it displayed on the page (we can decide what we want to display among the various Spoonacular options). From the same page, after a choice is made, we could have a button appear with "find matching recipes" (or something similar)
- page "recipes" - I am calling it recipes because the wine pairing is an option of the same spoonacular API so we said we would display recipes fetched from another API (Edamam is the one for which I have created an account). For now, I imagine we need to manually match in JS one or more recipes to each wine available in our select list. For now we have 15 wines listed for each wine type, so we can decide to reduce that number if needed. Of course let me know if you guys come up with a better idea for the use of the second API.